src/main/java/net/vulkanmod/vulkan/VRenderSystem.java

```diff
@@ -163,6 +171,40 @@ public static void depthMask(boolean b) {
         depthMask = b;
     }
 
+    public static void setPrimitiveTopologyGL(final int mode) {
+        switch (mode) {
+            case GL11.GL_LINES -> {
+                VRenderSystem.topology = VK_PRIMITIVE_TOPOLOGY_LINE_LIST;
+            }
+            case GL11.GL_LINE_STRIP -> {
+                VRenderSystem.topology = VK_PRIMITIVE_TOPOLOGY_LINE_STRIP;
+            }
+            // TODO: Use Triangle Fan topology if supported by the device
+            case GL11.GL_TRIANGLE_FAN, GL11.GL_TRIANGLES -> {
+                VRenderSystem.topology = VK_PRIMITIVE_TOPOLOGY_TRIANGLE_LIST;
+            }
+            case GL11.GL_TRIANGLE_STRIP -> {
+                VRenderSystem.topology = VK_PRIMITIVE_TOPOLOGY_TRIANGLE_STRIP;
+            }
```

Comment:
Is there a performance benefit to doing this? I did this because it's what Minecraft does.

Reply:
Less pipelines and less state changes. Probably there is not much difference in most scenarios (not many topology changes). Though it's probably worth since it's not that big of a change.

Reply:
So, you want me to change it back to emulating using triangle list, correct?

Reply:
Done.